provision/acc_provision/acc_provision.py

```diff
@@ -915,6 +915,11 @@ def Raise(exception):
         else:
             extra_checks = {}
     elif is_calico_flavor(config["flavor"]):
+        if config["aci_config"]["cluster_l3out"].get("vrf") is not None:
```

Comment:
Ok, if only cluster_l3out tenant is mentioned and not cluster_l3out vrf, then we can generate different vrf that we want to create as the tenantname_vrf. ( instead of checking-this is a change from the suggestion earlier). Of course if user mentions the vrf also under cluster_l3out, then we use it.  On the other hand, if only cluster_l3out vrf is mentioned, then we assume same tenant as external_l3out.